sql/driver/conn.go

```diff
@@ -77,8 +77,12 @@ func (c *conn) Query(stmt string, args []driver.Value) (*rows, error) {
 		}
 		params = append(params, param)
 	}
+	// Forget the session state, and use the one provided in the server
```

Comment:
On Tue, Aug 18, 2015, 10:03 PM Tamir Duberstein <notifications@github.com>
wrote:

In sql/driver/conn.go
<https://github.com/cockroachdb/cockroach/pull/2150#discussion_r37373867>:

> @@ -77,8 +77,12 @@ func (c *conn) Query(stmt string, args []driver.Value)
(*rows, error) { > } > params = append(params, param) > } > + // Forget the
session state, and use the one provided in the server

how come the "forgetting" part is necessary?

The forgetting part is not necessary. But I think what makes sense here is
for me to add a panic on the session update side to check that the client
session is nil.


—
Reply to this email directly or view it on GitHub
<https://github.com/cockroachdb/cockroach/pull/2150/files#r37373867>.<img
src="
https://ci6.googleusercontent.com/proxy/uJQ103R_qzNhcffPpGWyKl7dgNA-9m1rXhjjiIiulodCy49DROTEHsPMmKjXz_cRaSrhQGlcXXOAA277O9LuvNB3KYSdG6oGjkrknURqwBkQElzNP3iUlCNQVv_ZQJUYT9Zonf7AE2LnDrwbyl0fj9c-JR9J5Q=s0-d-e1-ft#https://github.com/notifications/beacon/ALOpBGB1ekNRWBZjMolx7r2idMJo26iDks5oo9uEgaJpZM4Ft2wM.gif
">